src/functions/env.ts

```diff
@@ -0,0 +1,229 @@
+import * as fs from "fs";
+import * as path from "path";
+
+import { FirebaseError } from "../error";
+import { logger } from "../logger";
+
+const RESERVED_KEYS = [
+  // Cloud Functions for Firebase
+  "FIREBASE_CONFIG",
```

Comment:
Just found out that we also support CLOUD_RUNTIME_CONFIG as a way of passing Runtime Config data. We should disallow that one as well.